### PR TITLE
functionality:add new fields to edit lifebank profile

### PR DESCRIPTION
**Pull request by AngeloCG97**
## Lifebank info pages 

#186 

### What does this PR contain?
- add new fields to section edit profile lifebank
- add function to load schedule to edit the profile of a lifebank

### How this should be tested?
- make run
- go to http://localhost:3000

### CODE REVIEW

File: hapi/src/api/account.api.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const historyApi = require('./history.api')
 const notificationApi = require('./notification.api')
 const userApi = require('./user.api')
 const vaultApi = require('./vault.api')
-const LIFEBANKCODE_CONTRACT = eosConfig.lifebankCodeContractName
+const LIFEBANKCODE_CONTRACT = 'lifebankcode' // @todo: use ENV
```
Comment (JustinCast) [inline hapi/src/api/account.api.js:14]: Take a look at codefactor warning

File: webapp/src/components/Schedule/Schedule.js [hunk 2]
```diff
@@ -233,7 +238,7 @@ const Schedule = ({ handleOnAddSchedule, data, showSchedule, showButton }) => {
         aria-describedby="transition-modal-description"
         className={classes.modal}
         open={openModal}
-        onClose={() => {}}
+        onClose={() => { }}
```
Comment (JustinCast) [inline webapp/src/components/Schedule/Schedule.js:241]: What about this empty handling?

File: webapp/src/routes/EditProfile/EditProfileBank.js [hunk 3]
```diff
@@ -84,7 +86,7 @@ const useStyles = makeStyles((theme) => ({
   }
 }))
 
-const EditProfileBank = ({ profile, onSubmit, loading }) => {
+const EditProfileBank = ({ profile, onSubmit, setField, loading }) => {
   const classes = useStyles()
   const [user, setUser] = useState({
     description: profile.description,
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileBank.js:92]: Take a look at codefactor's warnings

**Review by JustinCast — CHANGES_REQUESTED** [review]

**Review by JustinCast — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**